cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/CloudEntityResourceMapper.java

```diff
@@ -183,6 +187,7 @@ private static UUID parseGuid(String guid) {
         try {
             return UUID.fromString(guid);
         } catch (IllegalArgumentException e) {
+            LOGGER.warn(MessageFormat.format("Could not parse GUID string [{0}]:", guid), e);
```

Comment:
The formatting [{0}] is different from the one that we are using \"{0}\". Is it on purpose or ?

Reply:
Nope, by accident. I'll change it.